communication/marketing-team/role-handbooks/Events.md

```diff
@@ -0,0 +1,42 @@
+# Event Co-ordinator Role Handbook 
+
+## Overview
+
+As Event Co-ordinator Lead, you are responsible for
+- connecting with organizers of contributor events such as the monthly community meeting or contributor summits
```

Comment:
The contributor summit staff usually has an assigned comms team.  Would this replace that?

Also, what about external community events, such as Kubernetes Community Days?

Reply:
My intention for this role is that it would be a single point of contact on our side to ensure event-related comms are being handled. Generally I would expect the person in this role to be the comms team point of contact for the comms lead of events like the contributor summit. This person could also make sure that the event-specified "comms lead" is granted appropriate access to our usual tooling if they wanted to handle those things themselves. Essentially this person would act as the shepherd to make sure any needs of such events from our team, are being met. External community events are a very interesting point that we haven't really discussed. I'll bring that up in our next meeting.